cmd/laas/main.go

```diff
@@ -46,8 +51,25 @@ func main() {
 		log.Fatalf("Error loading .env file")
 	}
 
+	if os.Getenv("TOKEN_HOUR_LIFESPAN") == "" || os.Getenv("API_SECRET") == "" || os.Getenv("DEFAULT_ISSUER") == "" {
+		log.Fatal("Mandatory environment variables not configured")
+	}
+
```

Comment:
Should we move this below `flag.Parse()`. I do not think there is any functional difference, but it will keep the related blocks together.